js/tests/openapi/swagger.json

```diff
@@ -0,0 +1,974 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+      "title": "Immersive Reader Cognitive Service API",
+      "description": "Immersive Reader Cognitive Service API",
+      "version": "v1"
+    },
+    "paths": {
+      "/api/v1.0/GetContentModel": {
+        "post": {
+          "tags": [
+            "Api"
+          ],
+          "requestBody": {
+            "content": {
+              "application/json-patch+json": {
```

Comment:
we should remove all unsupported mime types from the Swagger, we only support **application/json** on each of these endpoints